src/main/scala-templates/limited/com/normation/plugins/datasources/EnablePluginImpl.scala

```diff
@@ -83,35 +82,33 @@ final class CheckRudderPluginDatasourcesEnableImpl() extends CheckRudderPluginDa
     } yield {
       (checked, version)
     }
+  } 
 
-  maybeInfo.fold( error => DataSourceLogger.error(error) , ok =>  DataSourceLogger.warn("License signature is valid.") )
+  //log at that point is we read the license information for the plugin
+  maybeInfo.fold( error => DataSourceLogger.error(error) , ok =>  DataSourceLogger.info("Plugin datasource has a license and the license signature is valid.") )
```

Comment:
Datasource plugin no ?

Reply:
I wanted to write "Plugin '${plugin name}' actually :)